src/NServiceBus.Core/Logging/StringBuilderExtensions.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace NServiceBus
+{
+    using System;
+    using System.Text;
+
+    static class StringBuilderExtensions
+    {
+        /// <summary>
+        /// Appends a new line marker to the StringBuilder followed by the text in the <paramref name="newLine"/>.
+        /// </summary>
+        public static StringBuilder NewLine(this StringBuilder stringBuilder, string newLine)
+        {
+            return stringBuilder.Append(Environment.NewLine).Append(newLine);
```

Comment:
Would it be better to use a single append with a format?
```suggestion
            return stringBuilder.AppendFormat("{0}{1}", Environment.NewLine, newLine);
```

Reply:
@timbussmann @danielmarbach do you know the consequence of using or the other? My naive understanding is that double appends result in less allocations.

Reply:
not sure, really but I think it won't matter as long as we're not using something like `Append(string.Format(...))`, so +1 for keeping as is, it's more readable imo.